homeassistant/config.py

```diff
@@ -627,7 +626,7 @@ def _identify_config_schema(module: ModuleType) -> \
     except (AttributeError, KeyError):
         return None, None
     t_schema = str(schema)
-    if t_schema.startswith('{'):
+    if t_schema.startswith('{') or 'schema_with_slug_keys' in t_schema:
```

Comment:
So hacky.

Reply:
I hear you... merge packages has always been hacky